dashboard/app/models/section.rb

```diff
@@ -53,12 +53,25 @@ class Section < ActiveRecord::Base
 
   LOGIN_TYPE_PICTURE = 'picture'.freeze
   LOGIN_TYPE_WORD = 'word'.freeze
+  LOGIN_TYPE_GOOGLE_CLASSROOM = 'google_classroom'.freeze
```

Comment:
Is it worth defining `LOGIN_TYPES` and validating that `login_type` is amongst this list?